public/app/core/components/AppChrome/MegaMenu/MegaMenuHeader.tsx

```diff
@@ -27,17 +25,26 @@ export function MegaMenuHeader({ handleMegaMenu, handleDockedMenu, onClose }: Pr
   return (
     <div className={styles.header}>
       <Stack alignItems="center" minWidth={0} gap={0.25}>
-        <ToolbarButton
+        { // begin NI fork changes
```

Comment:
nit: we've generally left this comment like `NI fork: <reason for change>`

Reply:
Handling in local `main` branch.